pkg/kubeapiserver/admission/configuration/configuration_manager.go

```diff
@@ -81,7 +89,38 @@ func (a *poller) notReady() {
 	a.ready = false
 }
 
+func (a *poller) bootstrapping(killTime func()) {
+	func() {
```

Comment:
Seems like an awkward structure - would recommend organizing this with the configuration and doing it under the same structure.